.gitignore

```diff
@@ -136,3 +136,6 @@ tags
 # examples
 examples/*/data/
 examples/*/visualizations/
+
+# benchmarking configs
+ludwig/benchmarking/configs/
```

Comment:
Is this right?

Reply:
It's working as expected on my local so I believe so